Formula/pcre2.rb

```diff
@@ -1,14 +1,10 @@
 class Pcre2 < Formula
   desc "Perl compatible regular expressions library with a new API"
   homepage "https://www.pcre.org/"
+  url "https://github.com/PhilipHazel/pcre2/releases/download/pcre2-10.40/pcre2-10.40.tar.bz2"
+  sha256 "14e4b83c4783933dc17e964318e6324f7cae1bc75d8f3c79bc6969f00c159d68"
   license "BSD-3-Clause"
 
-  # Remove `stable` block next release when patches are no longer needed
-  stable do
-    url "https://github.com/PhilipHazel/pcre2/releases/download/pcre2-10.40/pcre2-10.40.tar.bz2"
-    sha256 "14e4b83c4783933dc17e964318e6324f7cae1bc75d8f3c79bc6969f00c159d68"
-  end
-
   livecheck do
     url :stable
```

Comment:
after the change above, this (the url line) reference is stale and should be removed IMHO

Reply:
FWIW the livecheck method works without ANY url entry, since there is one from the package that can be used, and does result IMHO in the same logic.